kernelci/build.py

```diff
@@ -666,7 +668,7 @@ def get_default_output_path(cls, kdir):
         return os.path.join(kdir, 'build')
 
     @classmethod
-    def get_install_path(cls, kdir=None, output_path=None):
+    def get_install_path(cls, kdir=None, output_path=None, install_path=None):
         """Get the default build install path
```

Comment:
This needs to be updated as it's not just about getting the default install path.  Also the new `install_path` argument needs to be documented.

Another way to implement this would be to not add `install_path` and not return straight away when it's `None`, but rename this method to `.get_default_install_path()` and in the constructor do something like this:
```python
self._install_path = install_path or self.get_default_install_path(kdir, self._output_path)
```
and same thing in `kci_build`.  Part of this starts to look a bit clunky though with a `@classmethod` but the whole thing should be redesigned as job templates anyway so I don't know if it's worth making it nicer.  At least it should be consistent with the docstring.

Reply:
It's already there.  Did you mean something slightly different?

Reply:
I meant the changes you suggested above.

Reply:
...which are just about how to pass the default value around.  But actually I think this is all more of a distraction, whatever works from a functional point of view is good at this stage.